### DISCUSSION TITLE
Error: TypeError: null is not an object (evaluating '_reactNative.NativeModules.RNRazorpayCheckout.open')

### DISCUSSION THREAD

**Discussion by leamandeep** [expo/expo#22968] [Archived: Questions]
I am trying to integrate Razorpay in my expo app. I followed their docs but I'm stuck at this error. I asked in react community how to fix they told me to us a custom development client. I want to know from u how to fix it? I'm using expo go.

Here's the razorpay docs https://github.com/razorpay/react-native-razorpay#note-for-expo-applications

**Comment by tsapeta** [comment]
Razorpay has its own native code which is not included in Expo Go (it has a limited set of modules available). If you want to use this package, you'll need to use development builds or build the project locally. See our docs page: [Create a development build](https://docs.expo.dev/develop/development-builds/create-a-build/)
